src/main.js

```diff
@@ -74,14 +75,14 @@ p5.prototype.getMasterVolume = function () {
  *
  *  <p>If no value is provided, returns a Web Audio API Gain Node</p>
  *
- *  @method  masterVolume
+ *  @method  outputVolume
  *  @param {Number|Object} volume  Volume (amplitude) between 0.0
```

Comment:
volume is repeated 2 times

Reply:
yep, this is intentional because the first one gets parsed as the name, the second as part of the description by [jsdoc](https://jsdoc.app/tags-param.html)
<img width="750" alt="jsdoc name and description" src="https://user-images.githubusercontent.com/504124/119591700-f39a1f80-bd8b-11eb-9300-b765d4b1d4f0.png">